app/components/ui/SiteSearch.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FormEvent, useState } from "react";
+import { useHistory } from "react-router-dom";
+import cn from "classnames";
+import qs from "qs";
+import styles from "./SiteSearch.module.scss";
+
+// TODO: Previously used as a global search in SF Service Guide. Can be repurposed for Our415 as a Top Level Category
+// search per designs or similar. See:
+// https://www.figma.com/proto/iDBeNK7SnZXF8wc3pbPTCY/Our415-Site?node-id=2629-11357&t=q9JDI7LISYb2Ok6T-1
```

Comment:
Just FYI this figma link does have the site search in the nav, not sure if it's outdated

Reply:
Oh you're right. It looks like that site search shows up in both . I didn't even remember that. Or it was superseded in my mind as I was looking at the designs below. I asked Kyle again: https://www.figma.com/design/iDBeNK7SnZXF8wc3pbPTCY?node-id=3191-7191#855835281